src/assets/scss/components/dropdown.scss

```diff
@@ -3,29 +3,20 @@
   display: inline-block;
   padding: 0px 8px;
   position: relative;
-
-  .eos-icons {
+  
+  .eos-icons1 {
```

Comment:
don't change the the class name and please don't remove the properties either. The `eos-icons` class is the default for the icons.

Additionally, never use numbers in your classes, this is a very bad practice. Classes need to reflect their utility, not a version control system for classes.